resources/views/auth/forgot-password.blade.php

```diff
@@ -0,0 +1,25 @@
+<x-guest-layout>
+    <div class="mb-4 text-sm text-gray-600 dark:text-gray-400">
+        {{ __('Forgot your password? No problem. Just let us know your email address and we will email you a password reset link that will allow you to choose a new one.') }}
+    </div>
+
```

Comment:
Doet het niet bij mij, geeft een foutmelding: View [layouts.guest] not found.

Reply:
fixed